WalletWasabi.Fluent/CrashReport/ViewModels/CrashReportWindowViewModel.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.IO;
+using ReactiveUI;
+using System.Windows.Input;
+using Avalonia;
+using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Fluent.ViewModels;
+using WalletWasabi.Fluent.ViewModels.HelpAndSupport;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Fluent.CrashReport.ViewModels
+{
+	public class CrashReportWindowViewModel : ViewModelBase
+	{
+		public CrashReportWindowViewModel(SerializableException serializedException)
```

Comment:
Warning on `SerializedException` which is weird, it looks initialized for me.

Reply:
no warnings on my end